test.cpp

```diff
@@ -1,10 +1,12 @@
-#include <catch.hpp>
-#include <strong_type.hpp>
 #include <iomanip>
 #include <unordered_set>
 #include <memory>
 #include <algorithm>
 
+#include <catch.hpp>
+
+#include <strong_type/strong_type.hpp>
```

Comment:
I think `<strong_type/strong_type.hpp>` should be included 1st, to ensure there aren't any unmet header dependencies.